k8s/arm/README.md

```diff
@@ -0,0 +1,128 @@
+# Building OpenEBS for ARM
+
+OpenEBS runs in user space and does not have any Linux kernel module dependencies. This makes OpenEBS an ideal candidate for easily generating binaries and container images for ARM machines. The community is helping with refactoring the OpenEBS build scripts to make it easy to generate ARM binaries and container images. The efforts are lead by @Wangzihao18 @kmova. If you would like to help out with these efforts please reach out over [OpenEBS Slack #arm](https://openebs-community.slack.com/) channel. 
+
+ARM support is still under active development. As of 1.4.0, you can run OpenEBS Jiva and Local PVs on ARM. The steps below provide instructions on how to build the components for Jiva and Local PVs. 
+
+## Prerequisites
+
+The steps provided here assume that you have a ARM node that has the following installed:
+- Go 1.12.5 or higher
+- Docker 17.09 or higher
+
+Some of the below components might require additional dependencies to be installed. 
+
+## Building NDM
+
+NDM uses cgo to interact with libudev and seachest libraries for discovering device attributes. Prior to building NDM, setup the build tools using the instructions mentioned [here](https://github.com/openebs/node-disk-manager#build-image).
+
+NDM images can be built using the following. You can change the default container used for NDM using the `BASE_DOCKER_IMAGEARM64` ENV variable.
+
+```
+cd $GOPATH/src/github.com/openebs
+git clone https://github.com/openebs/node-disk-manager.git
+export BASE_DOCKER_IMAGEARM64=arm64v8/ubuntu:18.04 
+make
+```
+
+The above steps will create the following docker images:
+- openebs/node-disk-operator-arm64:ci
+- openebs/node-disk-manager-arm64:ci
+
+Tag the images and push to your repo. The following steps are used to push the alpha versions of this image to openebs docker repo. 
+
+```
+sudo docker tag openebs/node-disk-operator-arm64:ci openebs/node-disk-operator-arm64-ci:v0.4.4
+sudo docker tag openebs/node-disk-manager-arm64:ci openebs/node-disk-manager-arm64-ci:v0.4.4
+sudo docker push openebs/node-disk-operator-arm64-ci:1.4.0
```

Comment:
This [operator yaml](https://github.com/openebs/charts/blob/master/docs/openebs-operator-arm-dev.yaml) has the image tags v0.4.4 for NDM as per retag. I guess we need to change the push command also to same instead of 1.4.0